code/modules/client/verbs/suicide.dm

```diff
@@ -173,4 +173,5 @@
 		return
 	if(has_brain_worms())
 		src << "You can't commit suicide while you're controlling your host!"
```

Comment:
You should flavor this a bit. Say that the host suddenly begins to strongly resist you, similar to how 
**DEAD SPACE 2 SPOILER WARNING!!!!**
Isaac Clarke resists Marker Nicole when she tries to get him to kill himself, but was compliant with her suggestions right up to that point.